exercises/react/.meta/plugins.py

```diff
@@ -0,0 +1,20 @@
+def add_callbacks(case):
+    observers = []
+    callbacks = []
+    operations = list(filter(lambda op: op["type"] == "add_callback", case["input"]["operations"]))
+
+    if len(operations) == 0:
+        return '', {}
+
+    sufix = ''
+    if len(operations) > 1:
+        sufix = 'cb{i}_'
+    for i, op in enumerate(operations):
+        callbacks.append(op["name"])
+        observers.append(f'{sufix.format(i=i + 1)}observer')
```

Comment:
Using a **str.format** call inside an f-string seems just plain weird to me ... BTW, couldn't this whole thing be:

```python
for i, op in enumerate(operations):
    callbacks.append(op["name"])
    observers.append(f"cb{i + 1)" if i else "" + "observer")
```
?

Reply:
Yep, I did something similar to that :)